Source/driver/Castro_advance_ctu.cpp

```diff
@@ -129,14 +128,12 @@ Castro::do_advance_ctu(Real time,
 
       construct_ctu_hydro_source(time, dt);
       apply_source_to_state(S_new, hydro_source, dt, 0);
-      int is_new = 1;
-      clean_state(is_new, 0);
+      clean_state(S_new, cur_time, 0);
     }
 
 
     // Sync up state after old sources and hydro source.
-    int is_new=1;
-    frac_change = clean_state(is_new, Sborder, 0);
+    frac_change = clean_state(S_new, cur_time, 0);
```

Comment:
we were doing Sborder here before -- isn't that a different time?

Reply:
In this call, Sborder was the "old" state passed to enforce_min_density so that we could judge how bad the negative density was, if we had one. The actual cleaning was still on S_new.